perun-engine/src/main/java/cz/metacentrum/perun/engine/main/EngineStarter.java

```diff
@@ -41,11 +41,11 @@ public static void main(String[] args) {
 			engineStarter.springCtx.registerShutdownHook();
 			// Yes, we do this in the main thread because we do want to carry
 			// this initial steps out sequentially.
-			log.info("Gonna start Messaging.");
+			log.debug("Gonna start Messaging.");
 			engineStarter.engineManager.startMessaging();
-			log.info("Starting the runners.");
+			log.debug("Starting the runners.");
 			engineStarter.engineManager.startRunnerThreads();
-			log.info("Done. Perun-Engine has started.");
+			log.debug("Done. Perun-Engine has started.");
```

Comment:
I would keep at least this message in info. So when you start the engine, you can see in a log file, that it started.